Formula/cmctl.rb

```diff
@@ -0,0 +1,27 @@
+class Cmctl < Formula
+  desc "Command-line tool to manage cert-manager"
+  homepage "https://cert-manager.io"
+  url "https://github.com/cert-manager/cert-manager.git",
+    tag:      "v1.10.0",
+    revision: "da3265115bfd8be5780801cc6105fa857ef71965"
```

Comment:
```suggestion
      tag:      "v1.10.0",
      revision: "da3265115bfd8be5780801cc6105fa857ef71965"
```
Align the arguments to `url`.